onboarding/matplot.py

```diff
@@ -5,3 +5,25 @@
 - Be careful of data points that don't exist
 - Will require reading from CSV, can experiment with pandas
 '''
+
+import pandas
```

Comment:
nitpick: we generally import `pandas` as `pd` (it's the same effect)

Reply:
that really is a nitpick isn't it

almost half a nitpick really